native/sapp-windows/src/lib.rs

```diff
@@ -1023,21 +1087,32 @@ unsafe fn create_window() {
         right: 0,
         bottom: 0,
     };
+
     if _sapp.desc.fullscreen {
         win_style = WS_POPUP | WS_SYSMENU | WS_VISIBLE;
         rect.right = GetSystemMetrics(SM_CXSCREEN);
         rect.bottom = GetSystemMetrics(SM_CYSCREEN);
     } else {
-        win_style = WS_CLIPSIBLINGS
+        win_style = if _sapp.desc.window_resizable {
```

Comment:
Maybe do keep the common window flags outside of if? 

Something like this: 
win_style = WS_CLIPSIBLINGS | WS_CLIPCHILDREN | WS_CAPTION ..;
if _sapp.desc.window_resizable {
  win_style |= WS_SIZEBOX
}

Reply:
good callout! I can update this